index.html

```diff
@@ -13461,6 +13453,83 @@ <h2>Including Elements in the Accessibility Tree</h2>
             </li>
           </ul>
 	</section>
+	<section id="tree_relationships">
+	  <h2>Relationships in the Accessibility Tree</h2>
+          <p>The following terms are used to describe relationships between <abbr title="Document Object Model">DOM</abbr> elements.</p>
+          <p>The <dfn data-export="" data-lt="accessibility child|owned child|child element|child|children|child elements">accessibility children</dfn> of a DOM element are all of the children of that element's corresponding <a>accessible object</a> in the <a>accessibility tree</a>. In terms of the DOM, that includes the following:</p>
+          <ul>
+            <li>The <abbr title="Document Object Model">DOM</abbr> children of the <a>element</a>.
+            </li>
+            <li>All DOM descendants of the <a>element</a> with only elements of role <rref>generic</rref> or <rref>none</rref> intervening.
+            </li>
+            <li>All DOM children specified via an <pref>aria-owns</pref> relationship to the element.
```

Comment:
```suggestion
            <li>All DOM elements specified via an <pref>aria-owns</pref> relationship to the element.
```

Would it be accurate to say “elements” here instead of “children”?

Reply:
nice catch!